cmd/rpcdaemon/README.md

```diff
@@ -103,7 +103,7 @@ Example request
 `http POST http://localhost:8545/health --raw '{"min_peer_count": 3, "known_block": "0x1F"}'`
 Example response
 
-```
```

Comment:
unrelated change which was needed to invoke make docker image workflow (which didn't get invoked only on commit changes in .github/ dir)